app/views/editions/show.html.erb

```diff
@@ -49,6 +49,24 @@
         <%= @edition.updated_at %>
       </div>
     </div>
+
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        Authors
```

Comment:
I followed examples on http://getbootstrap.com/components/#panels. These are very likely to change once Mike goes through the UI (soon), so I wouldn't worry about it too much.